gen/xbuild.sh

```diff
@@ -8,51 +8,75 @@
 # compressed in an archive form (`.zip` for windows and `.tar.gz` for the rest)
 # within a folder named `dist`.
 
-set -o verbose
-
 [ -z $version ] && version=$(git describe --tags)
 
-mkdir -p dist
+if test -e lf || test -e lf.exe; then
+  echo "gen/xbuild.sh: delete lf and lf.exe from cwd before running this script"
+  exit 2
+fi
```

Comment:
I think it would be better if you just remove these files unconditionally using `rm -f lf lf.exe`

Reply:
I considered that, but I wouldn't expect `gen/xbuild.sh` to overwrite `lf` if I didn't know exactly what the script does. Overwriting `lf` silently could make the user unexpectedly lose information they might not be able to easily recover (if they ran `go build` before modifying something, say).

Reply:
I guess there are two approaches to this:

1. Check for files that shouldn't exist, and exit if they do
2. Nuke everything and proceed

I lean towards the latter approach, but this is a probably a matter of taste. This was just a suggestion and I don't mind too much either way.

Reply:
I'm thinking of rewriting things to make the compiler put the compiled binaries in `dist/`. This makes the code slightly more complicated.

I'd probably just nuke dist/. As I said in the other comment, it's owned by `xbuild.sh` AFAIC.